sched/signal/sig_findaction.c

```diff
@@ -23,8 +23,11 @@
  ****************************************************************************/
 
 #include <nuttx/config.h>
+
 #include "signal/signal.h"
 
+#include <nuttx/sched.h>
```

Comment:
move before line 27

Reply:
Oh, I missed this one. Thanks, @xiaoxiang781216!